src/json-patch-ot-agent.js

```diff
@@ -73,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
+	this.ackLocalVersion = newState[this.localPath.replace(/^\//, '')];
```

Comment:
This will not access nested versions, like '/path/to/meta/localVersion' (`localPath` is standardized JSON Pointer path, not just property name)